pkg/glusterfs/api/types.go

```diff
@@ -28,7 +28,7 @@ const (
 	EntryStateUnknown EntryState = ""
 	EntryStateOnline  EntryState = "online"
 	EntryStateOffline EntryState = "offline"
-	EntryStateFailed  EntryState = "failed"
+	EntryStateFailed  EntryState = "removed"
```

Comment:
True. I think for that reason, this PR cannot be changed. (API: https://github.com/heketi/heketi/blob/master/pkg/glusterfs/api/types.go#L31 ).

Instead we can always change the string shown in the CLI